source/unstyled.html.erb

```diff
@@ -70,6 +70,16 @@
     <div class="refill-snippet"><%= code_for("unstyled/footer") %></div>
   </section>
 
+  <section id="search-bar" class="refill">
+   <div class="refills-wrapper">
+     <div class="line-behind-text"><h6>Search bar</h6></div>
```

Comment:
Maybe we can do a PR for all of these after the unstyled elements are all merged?